blockchains/ethereum/tokenlist.json

```diff
@@ -13,9 +13,6 @@
             "decimals": 18,
             "logoURI": "https://assets.trustwalletapp.com/blockchains/ethereum/assets/0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2/logo.png",
             "pairs": [
```

Comment:
can you delete `pairs` for ethereum like chains?

Reply:
Done.  Note that WETH is moved later in the list, because ordering was based on the size of the pairs[] (descending) anf then by assetId.  Pairs is missing, so sorting is now only based on assetId.